content/news/call-for-sponsors.md

```diff
@@ -0,0 +1,27 @@
++++
+title = 'Call for Sponsors'
+date = 2024-08-16T20:26:21-05:00
+draft = false
+meta_title = "Help support this year's Global PyLadies Conference"
+description = "Help support this year's Global PyLadies Conference"
+tags = ["python", "community", "conference", "sponsorship", "pyladies"]
+layout = "news"
+categories = ["Blog Post",]
+image = ''
+imagealt = ''
++++
```

Comment:
```suggestion
author: "Naomi" 
+++
```

I realized the `author` info is missing from the news front matter :( It's my mistake, I will fix it tomorrow but for now please add the author info accordingly.
Thank you.

Reply:
Okay, will do.